sdk/core/azure-core/inc/azure/core/internal/http/pipeline.hpp

```diff
@@ -150,6 +150,20 @@ namespace Azure { namespace Core { namespace Http { namespace _internal {
               clientOptions.Transport));
     }
 
+    [[deprecated]]
+    explicit HttpPipeline(
+        Azure::Core::_internal::ClientOptions const& clientOptions,
+        std::vector<std::unique_ptr<Azure::Core::Http::Policies::HttpPolicy>>&& perRetryPolicies,
+        std::vector<std::unique_ptr<Azure::Core::Http::Policies::HttpPolicy>>&& perCallPolicies)
+        : HttpPipeline(
+            clientOptions,
+            "security.attestation",
```

Comment:
If we need a placeholder, let's use something generic.

Also, add back the doc comments.

This is the ctor we are bringing back, correct? Can we add it back verbatim?

https://github.com/Azure/azure-sdk-for-cpp/blob/0e9fba2623a7c7d4625573e46809e7e358323be9/sdk/core/azure-core/inc/azure/core/internal/http/pipeline.hpp#L181-L206

Reply:
We don't want a placeholder. We want a constructor with 3 parameters that allows the attestation service to continue to work.

And I *don't* want documentation for this if at all possible. It's [[deprecated]] and nobody should be using it unless you're version 1.0.0 of the attestation service.

Reply:
Remember that the entire purpose of the 3 parameter constructor is that attestation 1.0 is using it. The problem is that the 3 parameter constructor doesn't have package name and version as parameters, so for services using the 3 parameter constructor, I need to somehow come up with the strings "security.attestation" and "1.0.0" so that the User-Agent header is correctly formed for the attestation service.